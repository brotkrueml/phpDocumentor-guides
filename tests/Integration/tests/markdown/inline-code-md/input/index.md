# Markdown with inline Code

Lorem Ipsum Dolor: `$dolor`...
