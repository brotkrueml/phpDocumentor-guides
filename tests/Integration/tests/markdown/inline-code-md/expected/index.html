<!DOCTYPE html>
<html lang="en">
<head>
    <title>Markdown with inline Code - Project Title</title>

</head>
<body>
    <h1>Markdown with inline Code</h1>

<p>Lorem Ipsum Dolor: <code>$dolor</code>...</p>
</body>
</html>
